steps/plandek-deployment/2.0.0/step.yml

```diff
@@ -0,0 +1,162 @@
+title: Plandek - deployment
+summary: |
+  Enables you to track and visualize build and deployment events within Plandek's metrics and dashboards.
+description: |
+  Wrapper implementing the Plandek /deployment POST API. Used to complete a deployment within Plandek for a given release cycle.
+  More details can be found here: [https://api.plandek.com/deployments/v1/ui/](https://api.plandek.com/deployments/v1/ui/)
+website: https://github.com/secretescapes/bitrise-step-plandek-deployment
+source_code_url: https://github.com/secretescapes/bitrise-step-plandek-deployment
+support_url: https://github.com/secretescapes/bitrise-step-plandek-deployment/issues
+published_at: 2023-08-30T10:32:24.454204+02:00
+source:
+  git: https://github.com/secretescapes/bitrise-step-plandek-deployment.git
+  commit: 7228ccb6d6a71141aa2ab534f2b7f6b86895e970
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  - name: curl
+  apt_get:
+  - name: jq
+  - name: curl
+is_always_run: true
+inputs:
+- api_token: null
+  opts:
+    description: |
+      Plandek API token.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: |
+      Plandek API token.
+    title: API token
+- client_key: null
+  opts:
+    description: |
+      Unique identifier of your instance.
+    is_required: true
+    summary: |
+      Unique identifier of your instance.
+    title: Client key
+- opts:
+    description: |
+      Typically corresponds to the name of your pipeline, workflow, repo, or service under which builds are compiled.
+      If you use the same name pipelines across your organization, we recommend appending additional information to
+      your pipeline name (e.g. repo name) to make it easier to understand your data.
+    is_expand: true
+    is_required: true
+    summary: |
+      Typically corresponds to the name of your pipeline, workflow, repo, or service under which builds are compiled.
+    title: Pipeline
+  pipeline: $BITRISE_APP_TITLE
+- build: null
+  opts:
+    description: |
+      Build identifier in your build tool. Typically the version being deployed (e.g. Svc B 10.2.1).
+    is_required: true
+    summary: |
+      Build identifier in your build tool. Typically the version being deployed (e.g. Svc B 10.2.1).
+    title: Build
+- commits: null
+  opts:
+    description: |
+      The full commit hash(es) of the release being deployed.
+    is_required: true
+    skip_if_empty: true
```

Comment:
why has this input been configured with `skip_if_empty` ? Is it differentiated in the code base when the env var has an empty value and when it is not set at all?